emmaa/statements.py

```diff
@@ -20,3 +20,11 @@ def __init__(self, stmt, date, search_terms):
     def __repr__(self):
         return '%s(%s, %s, %s)' % (self.__class__.__name__, self.stmt,
                                    self.date, self.search_terms)
+
+    def to_json(self, use_sbo=False):
+        # Get json representation of statement
+        json_stmt = self.stmt.to_json(use_sbo=use_sbo)
+        # Stringify source hashes: JavaScript can't handle int's of length > 16
+        for ev in json_stmt['evidence']:
+            ev['source_hash'] = str(ev['source_hash'])
+        return json_stmt
```

Comment:
For this method, would be good to wrap the internal INDRA Statement JSON in an enclosing dict that included the other EmmaaStatement metadata (date and search terms).

Reply:
Ah, instead of just returning the indra statement?

Reply:
Fixed in [this branch](https://github.com/kkaris/emmaa/blob/wrap_to_json/emmaa/statements.py#L24)